src/parse:properties.ts

```diff
@@ -132,11 +132,29 @@ if (!fs.existsSync(file)) {
             raw: true,
         })
             .then((data) => (data as Partial<PostcodeType>[]).forEach(({ postcode, ...v }) => postcodes.set(postcode, v))),
+        orm.Marker.findAll({
+            attributes: ['label'],
+            where: {
+                type: {
+                    //@ts-ignore
+                    [orm.Sequelize.Op.eq]: MarkerTypeEnum.property
+                }
+            },
+            raw: true,
+        })
+            .then((data) => (data as Partial<MarkerType>[]).forEach((v) => markersStore.add(v.label))),
         orm.Property.findAll({
```

Comment:
according logic i think there is no need to query `orm.Property` because there should not be Transaction without `Property` :)